TypeTreeGeneratorAPI/TypeTreeGenerator/AssetStudio/AssetStudioGenerator.cs

```diff
@@ -46,9 +46,10 @@ public override void LoadDll(Stream dllStream)
             return monoBehaviourDefs;
         }
 
-        public override List<TypeTreeNode>? GenerateTreeNodes(string assemblyName, string fullName)
-        {
-            var typeDef = GetTypeDefinition(assemblyName, fullName);
+        public override List<TypeTreeNode>? GenerateTreeNodes(string assemblyName, string fullName) {
```

Comment:
All good overall,
the formatting is just missing on the ``{`` here, which should be moved to a new line to be uniform with the remaining code.